examples/seismic/abc_methods/02_damping.ipynb

```diff
@@ -0,0 +1,1257 @@
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "graph2dvel(v0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Time parameters are defined and constructed by the following sequence of commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t0    = 0.\n",
+    "tn    = 1000.   \n",
+    "CFL   = 0.4\n",
+    "vmax  = np.amax(v0) \n",
+    "dtmax = np.float64((min(hxv,hzv)*CFL)/(vmax))\n",
+    "ntmax = int((tn-t0)/dtmax)+1\n",
+    "dt0   = np.float64((tn-t0)/ntmax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the temporal parameters, we generate the time informations with *TimeAxis* as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time_range = TimeAxis(start=t0,stop=tn,num=ntmax+1)\n",
+    "nt         = time_range.num - 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The symbolic values associated with the spatial and temporal grids that are used in the composition of the equations are given by:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(hx,hz) = grid.spacing_map  \n",
+    "(x, z)  = grid.dimensions     \n",
+    "t       = grid.stepping_dim\n",
+    "dt      = grid.stepping_dim.spacing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We chose a single Ricker source, whose frequency is $ 0.005Khz $. This source is positioned at $\\bar{x}$ = 35150m and $\\bar{z}$ = 32m. We then defined the following variables that represents our choice:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f0      = 0.01\n",
+    "nsource = 1\n",
+    "xposf   = 0.5*(compx-2*npmlx*hxv)\n",
+    "zposf   = hzv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we know, Ricker's source is generated by the *RickerSource* command. Using the parameters listed above, we generate and position the Ricker source with the following sequence of commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "src = RickerSource(name='src',grid=grid,f0=f0,npoint=nsource,time_range=time_range,staggered=NODE,dtype=np.float64)\n",
+    "src.coordinates.data[:, 0] = xposf\n",
+    "src.coordinates.data[:, 1] = zposf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below we include the plot of Ricker source."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "src.show()"
```

Comment:
Doesn't seem to be displaying correctly?

Reply:
The cell type in Jupyter Notebook was wrong. It's ok now!